.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -290,6 +290,16 @@
             <MauiImage Include="@(MauiIcon)" IsAppIcon="True" />
         </ItemGroup>
 
+        <!-- Add a hash to the splash screen so we can use it later -->
+        <GetFileHash Files="@(MauiSplashScreen)" HashEncoding="base64">
```

Comment:
The `HashEncoding` defaults to `hex`:

https://learn.microsoft.com/en-us/visualstudio/msbuild/getfilehash-task?view=vs-2022